docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
 
   backend:
-    image: 'clowder/clowder2-backend:2.0.0-beta.2'
+    image: 'clowder/clowder2-backend:release-v2.0-beta-3'
```

Comment:
We haven't officially released yet hence if you wish to use beta3 you have to use this image.
But once released, the tag change will be a part of the release PR.